src/components/common/MoviesList.tsx

```diff
@@ -1,41 +1,96 @@
-import axios from "axios";
 import { useEffect, useState } from "react";
+import Slider from "react-slick";
+import axios from "axios";
+
+function SampleNextArrow(props: any) {
+	const { className, style, onClick } = props;
+	return (
+		<div
+			className={className}
+			style={{ ...style, display: "block", right: "0px" }}
+			onClick={onClick}
+		/>
+	);
+}
+
+function SamplePrevArrow(props: any) {
+	const { className, style, onClick } = props;
+	return (
+		<div
+			className={className}
+			style={{
+				...style,
+				display: "block",
+				left: "0px",
+				zIndex: 1,
+			}}
+			onClick={onClick}
+		/>
+	);
+}
 
 export default function MoviesList() {
 	const [movies, setMovies] = useState<any[]>([]);
+	const [like, setLike] = useState<boolean>(false);
+
 	useEffect(() => {
 		axios
```

Comment:
create a seperate file for api call

export const getMovies = () => {
  const movies = []
     axios.get().then
     return movies}